keys/cmd/create.go

```diff
@@ -0,0 +1,163 @@
+package cmd
+
+// TODO: modify module structure to prevent crypto/ copy redundancy
+
+import (
+	"bufio"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/client/input"
+	"github.com/cosmos/go-bip39"
+	"github.com/spf13/cobra"
+	"github.com/syndtr/goleveldb/leveldb"
+	cryptoPocket "keys/crypto"
+)
+
+const (
+	flagRecover = "recover"
+)
+
+// createCmd represents the create command
+var createCmd = &cobra.Command{
```

Comment:
I'm not convinced the `CLI` portion of the keybase should be in this pacakge and not in the `app/client/cli` folder. WYT @Olshansk @deblasis